core/vm/logger.go

```diff
@@ -76,6 +77,7 @@ type StructLogger struct {
 	cfg LogConfig
 
 	logs          []StructLog
+	sizeLogs      int
```

Comment:
Any Go array's (`[]Type`) length can be obtained using the build in `len` function.